include/deal.II/base/types.h

```diff
@@ -56,13 +56,17 @@ namespace types
 
 #ifdef DEAL_II_WITH_64BIT_INDICES
   /**
-   * The type used for global indices of degrees of freedom. While in
-   * sequential computations the 4 billion indices of 32-bit unsigned integers
-   * is plenty, parallel computations using the
+   * The type used to denote the global index of degree of freedom. This
+   * type is then also used for querying the global *number* of degrees
+   * of freedom, since the number is simply the largest index plus one.
+   *
+   * While in sequential computations the 4 billion indices of 32-bit unsigned
+   * integers is plenty, parallel computations using the
    * parallel::distributed::Triangulation class can overflow this number and
```

Comment:
Since `parallel::distributed::Triangulation` is not the only class that can do this, should we phrase this more generally by `using, e.g., the parallel::distributed::Triangulation` class (here and below)?